### DISCUSSION TITLE
希望能改一下“删除视频弹窗”的提示文本

### DISCUSSION THREAD

**Discussion by ChurchEChan** [the1812/Bilibili-Evolved#1695] [功能建议 / Ideas]
目前的文本是“删除视频内弹出的三连提示框”，希望能加上弹幕投票，“删除视频内弹出的三连提示框和弹幕投票”。
刚才想把弹幕投票恢复，但是一时脑抽想不起是哪条设置，搜“弹幕弹窗”也搜不到，最后是一条条地找才找到的。如果能加上的话，会方便想要打开的人定位到设置。

**Comment by Geeyun-JY3** [comment]
确实：https://github.com/the1812/Bilibili-Evolved/discussions/1332#discussioncomment-306332
